mscore/notetweakerdialog.cpp

```diff
@@ -0,0 +1,164 @@
+//=============================================================================
```

Comment:
Now it needs to be moved to `mscore/pianoroll'

Reply:
Not just `notetweakerdialog.cpp`, but `notetweakerdialog.h` and `notetweakerdialog.ui` as well. Then these 3 files should be added to the appropriate sections of the `mscore/pianoroll/pianoroll.cmake` file, and references to these files should be removed from `mscore/CMakeLists.txt`, because they will be covered by `${PIANOROLL_SRC}` and `${PIANOROLL_UI}`.